validator/client/src/test/java/tech/pegasys/teku/validator/client/AttestationDutySchedulerTest.java

```diff
@@ -264,43 +175,14 @@ public void shouldDelayExecutingDutiesUntilSchedulingIsComplete() {
     dutyScheduler.onAttestationCreationDue(ZERO);
     dutyScheduler.onAttestationAggregationDue(ZERO);
     // Duties haven't been loaded yet.
-    verify(scheduledDuties, never()).produceBlock(ZERO);
     verify(scheduledDuties, never()).produceAttestations(ZERO);
     verify(scheduledDuties, never()).performAggregation(ZERO);
 
     epoch0Duties.complete(Optional.of(emptyList()));
-    verify(scheduledDuties).produceBlock(ZERO);
     verify(scheduledDuties).produceAttestations(ZERO);
     verify(scheduledDuties).performAggregation(ZERO);
   }
 
-  @Test
-  public void shouldNotPerformDutiesForSameSlotTwice() {
```

Comment:
This is an important test to keep for the attestation duty scheduler but will have to be converted to checking it doesn't publish attestations for the same slot rather than blocks in this case.